src/ws/session-utils.c

```diff
@@ -798,38 +798,6 @@ seal_memfd (FILE *memfd)
   return fcntl (fileno (memfd), F_ADD_SEALS, seals) == 0;
 }
 
-/* signal- and after-fork()-safe function to format a string, print it
- * to stderr and abort execution.  Never returns.
- */
-static noreturn void
-__attribute__ ((format (printf, 1, 2)))
-abort_with_message (const char *format,
```

Comment:
Put back this commit, split out the message printing part, and consistently replace all warn[x]() and err[x]() calls after fork with it.